package.json

```diff
@@ -249,6 +251,8 @@
     "react-hot-loader": "1.3.0",
     "react-test-env": "0.2.0",
     "readline-sync": "1.4.5",
+    "reason-js": "0.3.0",
```

Comment:
`reason-js` can be removed 🙂 The `bs-loader` docs were outdated and @chenglou recently updated them to match the current API and standards